my-app/src/components/blogPreview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import style from './blogPreview.module.css'
+import type { Blog } from '@/app/blogData'
+import Link from "next/link";
+
+
+
+export default function BlogPreview({title, date, description, slug, id} : Blog) {

```

Comment:
Nice job using a component with unique props here! You might want to consider making the styling for these components more clear. At first, it looked like one big preview (a box around each one would help clarify!).